src/problem1.js

```diff
@@ -1,6 +1,179 @@
+const INPUT = {
+  length: 2,
+  numType: 'number',
+  minPage: 3,
+  maxPage: 398,
+};
+
+const REDUCING = {
+  additionInit: 0,
+  multiplicationInit: 1,
+};
+
+const RESULT = {
+  invalidInput: -1,
+  draw: 0,
+  pobiWin: 1,
+  crongWin: 2,
+};
+
 function problem1(pobi, crong) {
-  var answer;
-  return answer;
+  if (!isValidInput(pobi) || !isValidInput(crong)) {
+    return RESULT.invalidInput;
+  }
+
+  return solution(pobi, crong);
```

Comment:
지금도 좋지만, soltution 로직이 problem1함수 안으로 들어오거나 isvalid로직을 한번 더 감싸주는게 좋을 것 같아요! 
problem 함수와 solution이라는 함수가 의미적으로 같은 느낌이라 계층을 분할하지 않는게 좋을 것 같아요!

```suggestion
validationCheck(pobi, crong);
return solution(pobi, crong);
```

혹은 
```suggestion
if (!isValidInput(pobi) || !isValidInput(crong)) {
    return RESULT.invalidInput;
  }
  const pobiScore = getScore(pobi);
  const crongScore = getScore(crong);
  return compareScore(pobiScore, crongScore);
```

이런식으로하면 코드의 추상화 계층을 맞춰줄 수 있을 것 같아요!

Reply:
와 빠른 리뷰 너무 감사드립니다!

"problem 함수와 solution이라는 함수가 의미적으로 같은 느낌이라 계층을 분할하지 않는게 좋을 것 같아요!"
라고 말씀하신걸 보고 제가 solution을 완성하라는 문장 자체에 너무 포커싱하고 있었단걸 깨달았습니다!

리뷰해주셔서 정말 중요한 부분을 체크하게 됐습니다.
2주차 과제부터 말씀해주신 부분 염두에 두고 진행하겠습니다! 감사합니다 ☺️